assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -167,6 +162,70 @@ const AdUnits = ( {
 					isSuccess
 				/>
 			) }
+			<div className="flex items-end" style={ { margin: '-30px 0' } }>
+				<TextControl
+					className="mr2"
+					label={ __( 'Network code', 'newspack' ) }
+					value={ networkCode }
+					onChange={ setNetworkCode }
+					disabled={ serviceData.status.connected }
+				/>
+				<Card buttonsCard noBorder>
+					<Button onClick={ saveNetworkCode } isPrimary disabled={ serviceData.status.connected }>
+						{ __( 'Save', 'newspack' ) }
+					</Button>
+					{ serviceData.status.connected && (
+						<>
+							<Button
+								onClick={ () => {
+									credentialsInputFile.current.click();
+								} }
+								isSecondary
+								isSmall
+							>
+								{ __( 'Upload new credentials', 'newspack' ) }
+							</Button>
+							<Button
+								onClick={ () => {
+									setIsRemoving( true );
+								} }
+								// isQuarternary
+								isSmall
+								isDestructive
+							>
+								{ __( 'Remove credentials', 'newspack' ) }
+							</Button>
+						</>
+					) }
+					{ isRemoving && (
+						<Modal
+							title={ __( 'Remove Service Account Credentials', 'newspack' ) }
+							onRequestClose={ () => setIsRemoving( false ) }
+						>
+							<p>
+								{ __(
+									'The credentials will be removed and Newspack Ads will no longer be connected to this Google Ad Manager account.',
+									'newspack'
+								) }
+							</p>
+							<Card buttonsCard noBorder className="justify-end">
+								<Button
+									isDestructive
+									onClick={ () => {
+										removeGAMCredentials();
+										setIsRemoving( false );
+									} }
+								>
+									{ __( 'Remove credentials', 'newspack' ) }
+								</Button>
+								<Button isSecondary onClick={ () => setIsRemoving( false ) }>
+									{ __( 'Dismiss', 'newspack' ) }
```

Comment:
Just for consistency with other Newspack UI:

```suggestion
									{ __( 'Cancel', 'newspack' ) }
```